CodeStarterCamp_Week1/STEP3.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Week1_STEP3.swift
+//  CodeStarterCamp_Week1
+//
+//  Created by Yelin Hong on 2022/02/13.
+//
+
+import Foundation
+//길이(Int)에 따른 막대(빼빼로 손잡이)를 그리는 함수
+func drawBar(barLength: Int) {
+    for _ in 1...barLength {
+        print(" | | ")
+    }
+}
+//몸통(String), 토핑(String)을 매개변수로 받아 토핑과 몸통을 그리는 함수
+func makeIngredient(main: String, topping: String) {
+    print(topping + main + topping)
+}
+
+//길이(Int)에 따라 몸통을 그리는 함수
+func drawBody(bodyLength: Int, main: String, topping: String) {
+    if main == "***" {
+        for _ in 1...bodyLength {
+            makeIngredient(main: main, topping: topping)
+        }
+    }
+    else {
+        for _ in 1...(bodyLength/2) {
+            makeIngredient(main: main, topping: topping)
+        }
+    }
```

Comment:
Swift 에서 공식적으로 StyleGuide를 제시해주진 않지만, 일반적으로 if에 else문이 붙으면 요렇게 작성합니다 🙂 
```suggestion
    } else {
        for _ in 1...(bodyLength/2) {
            makeIngredient(main: main, topping: topping)
        }
    }
```